platform/CircularBuffer.h

```diff
@@ -105,6 +105,24 @@ class CircularBuffer {
         core_util_critical_section_exit();
     }
 
+    /** Get the number of elements currently stored in the circular_buffer */
+    CounterType size() const
+    {
+        core_util_critical_section_enter();
+        CounterType elements;
+        if (!_full)
```

Comment:
Please use https://docs.mbed.com/docs/mbed-os-handbook/en/latest/cont/code_style/ (as it is in this file)